src/scripts.js

```diff
@@ -24,20 +25,32 @@ const recipeHeartSelector = document.querySelector('.heart-recipe');
 const calendarSelector = document.querySelector('.calendar');
 const recipeCalendarSelector = document.querySelector('.calendar-recipe');
 // const listCalendarSelector = document.querySelector('.calendar-list');
+const userGreeting = document.querySelector('.header__right--text');
+const userPantryList = document.querySelector('.pantry__bottom--left');
 
 let allRecipes;
+let currentUser;
 
 window.addEventListener('load', function() {
   console.log('page loaded 🥺');
   const recipeInstances = recipeData.map(recipe => new Recipe(recipe));
   allRecipes = new RecipeRepository(recipeInstances);
+
+  currentUser = new User(usersData.find(user => user.name === "Earline Von"));
+
+  // currentUser = new User(usersData[getRandomIndex(usersData.length)]);
```

Comment:
Looks like the user is hard-coded to be Earline (with her ample pantry) right now but can be random as soon as we uncomment this line. Nice!